src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -85,13 +85,23 @@ export default {
 
       return x !== prevX || y !== prevY;
     },
+    getCenterPosition() {
```

Comment:
Is there any way that this would be added to the shape class? Is that a class that we 'own'?

Reply:
Shape is part of JointJS; it's not something we own. The shape may return a JointJS "rect", in which case I could call `centre` on it: https://resources.jointjs.com/docs/jointjs/v3.0/geometry.html#g.Rect.prototype.center. I'll look into that.

I could also grab the position of the centre port and use that. But I'm not sure I want to couple this to the ports implementation.